src/signals/index.ts

```diff
@@ -1,3 +1,24 @@
-export function placeholder() {
-	return 'placeholder';
-}
\ No newline at end of file
+import { PartialStateUpdater, patchState, StateSignal } from '@ngrx/signals';
+import { immerReducer } from 'ngrx-immer';
+
+export type MutableStateUpdater<State extends object> = (state: State) => void;
```

Comment:
consider the following name:

```suggestion
export type ImmerStateUpdater<State extends object> = (state: State) => void;
```